### ISSUE TITLE
Chat Filter in whatsapp web 

### ISSUE THREAD

**Issue by PragneshAgola** [marcosvrs/WTF#30]
I am managing the history in the extension and I want to filter the chat list in whatsapp web on click of that history So How can I add the new Function for Filter chat in the code. Can you explain me How to Create the New function in code so I can Call that function from the extension and that function executes in whatsapp web.

**Comment by Himanshusha673** [comment]
yes i also wanted to make a crm feature like extension, but still finding a workaround for me
